addons/mail/static/src/views/activity/activity_controller.js

```diff
@@ -42,6 +35,16 @@ export class ActivityController extends Component {
         this.messaging = useMessaging();
         this.activity = useService("mail.activity");
     }
+    
+    get modelParams() {
+        return {
+            activeFields: this.props.archInfo.activeFields,
+            resModel: this.props.resModel,
+            fields: this.props.fields,
+            viewMode: "activity",
+            rootState: this.props.state || {},
```

Comment:
It should probably be: `this.props.state?.rootState` :thinking: 

or simply set this line in the getter:
`const { rootState } = this.props.state || {};`
then
```
return {
  ...
  rootState,
}
```